content/en/tutorials/cloud-pods-collaborative-debugging/index.md

```diff
@@ -0,0 +1,332 @@
+
+For developing AWS applications locally, the tool of choice is LocalStack, which can sustain a full-blown comprehensive stack.
+However, when issues appear, and engineers need a second opinion from a colleague, recreating the environment from scratch can leave
+details slipping through the cracks. This is where Cloud Pods come in, to encapsulate the state of the LocalStack instance and allow for seamless
+collaboration. While databases have snapshots, similarly, LocalStack uses Cloud Pods for reproducing state and data.
+
+In this tutorial, we will explore a common situation where a basic IAM misconfiguration causes unnecessary delays in finding the right solution.
+We will also discuss the best practices to prevent this and review some options for configuring Cloud Pod storage.
+The full sample application can be found [on GitHub](https://github.com/localstack-samples/cloud-pods-collaboration-demo) to clone, for following along more easily.
+
+### **Prerequisites**
+
+- [LocalStack CLI](https://docs.localstack.cloud/getting-started/installation/#localstack-cli) (preferably using `pip`)
+- [Docker](https://docs.docker.com/engine/install/)
+- [Terraform](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli) or [OpenTofu](https://opentofu.org/docs/intro/install/) and [terraform-local](https://docs.localstack.cloud/user-guide/integrations/terraform/#install-the-tflocal-wrapper-script)
+- Optional for Lambda build & editing: [Maven 3.9.4](https://maven.apache.org/install.html) & [Java 21](https://www.java.com/en/download/help/download_options.html)
+
+### Nice to have
+
+- Basic knowledge of AWS services (API Gateway, Lambda, DynamoDB, IAM)
+- Basic understanding of Terraform for provisioning AWS resources
+
+
+In this demo scenario, a new colleague, Bob, joins the company, clones the application repository, and starts working on the Lambda code. He will add the necessary 
+resources in the Terraform configuration file and some IAM policies that the functions need in order to access the database. 
+He is following good practice rules, where the resource has only the necessary permissions. However, Bob encounters an error despite this.
+
+### Architecture Overview
+
+The stack consists of an API Gateway that exposes endpoints and integrates with two Lambda functions responsible for adding and fetching
+products from a DynamoDB database. IAM policies are enforced to ensure compliance with the
+**[principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege)**, and the logs will be sent to the CloudWatch service.
+
+### Note
+
+This demo application is suitable for AWS and behaves the same as on LocalStack. You can try this out by running the Terraform configuration file against the AWS platform.
+
+![Application Diagram](cloud-pod-collab.png)
+
+### Starting LocalStack
+
+In the root directory, there is a `docker-compose.yml` file that will spin up version 3.3.0 of LocalStack, with an 
```

Comment:
Any reason for picking a specific version?

Reply:
it was the latest version at that time. if cloud pods change for future versions i won't go back to change this material